src/components/NewProjects/ProjectCard/ProjectCard.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+
+import "./ProjectCard.css";
+
+const ProjectCard = ({
+  title,
+  description,
+  image,
+  github,
+  website,
+  languages
+}) => {
+  return (
+    <article className="project-card">
+      <div className="project-card__title">
+        <div>{title}</div>
+      </div>
+      <div className="project-card__wrapper">
+        {image && <img src={image} alt="logo"/>}
+        <div>
+          <div className="project-card__description">
+            <div>{description}</div>
+          </div>
+          <div className="project-card__footer">
+            <div>
+              {github && (
```

Comment:
Il manque un petit espace entre `Repository` et `Website`. 
Qu'est-ce que vous pensez d'ajouter une icône Github et une icône "external website" ([exemple](https://image.flaticon.com/icons/svg/1/1424.svg)) à leur place?

Reply:
This is still a work in progress 😉